content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md

```diff
@@ -11,104 +11,233 @@ keywords:
 - data backup and recovery
 ---
 
-Google Photos works best in TrueNAS using a Google Photos API key and [rclone](https://rclone.org/) token.
+Configuring a Google Photos cloud sync task requires a Google Photos API key and [rclone](https://rclone.org/) token.
+See the [rclone Google Photos backend documentation](https://rclone.org/googlephotos/) for more on using rclone to sync Google Photos, including features and limitations of the Google Photos API.
+
+This is a multi-part procedure that includes [generating Google API credentials](#creating-the-api-credentials), [installing and configuring rclone](#configuring-rclone) on your client OS, [creating cloud credentials](#creating-google-photos-cloud-credentials) for Google Photos on TrueNAS SCALE, and then [configuring the cloud sync task](#creating-the-cloud-sync-task).
+
+## Before You Begin
+
+Review your storage and data protection requirements and consider your options before setting up a Google Photos cloud sync task.
+Consider how you want to manage your media files on Google Photos and in your local dataset.
+Select the cloud sync task direction, transfer mode, and remote folder to target that best fit your needs.
```

Comment:
Do they need to create their TrueNAS datasets before they try to complete this procedure?
If they need to take action to prepare the TrueNAS this is where we should say this, like we do when configuring an app like MinIO that needs certain datasets or some other configuration work in SCALE before they start the Google Photo/rclone process.